PowerPointLabs/PowerPointLabs/PictureSlidesLab/View/AdjustImageWindow.xaml.cs

```diff
@@ -256,5 +260,37 @@ private void ZoomOutButton_OnClick(object sender, RoutedEventArgs e)
         {
             _croppingAdorner.ZoomCroppingRect(-AdjustUnit);
         }
+
+        private void RotateFlipImg(RotateFlipType roatateFlipType)
+        {
+            var img = (Bitmap)Bitmap.FromFile(thumbnailImageFile.Text);
+            img.RotateFlip(roatateFlipType);
+            String rotatedImg = StoragePath.GetPath("crop-"
+                                    + DateTime.Now.GetHashCode() + "-"
+                                    + Guid.NewGuid().ToString().Substring(0, 7));
+            img.Save(rotatedImg);
+            thumbnailImageFile.Text = rotatedImg;
```

Comment:
then generate new thumbnail to `thumbnailImageFile.Text` (for display in the window) and `CropResultThumbnail` (one of the output)